src/main/java/net/bramp/ffmpeg/FFmpegUtils.java

```diff
@@ -58,7 +58,7 @@ public static String millisecondsToString(long milliseconds) {
    * @return
    */
   public static long parseBitrate(String bitrate) {
-    Pattern p = Pattern.compile("(\\d+(\\.\\d+)?)kbits/s");
+    Pattern p = Pattern.compile("(\\d+(?:\\.\\d+)?)kbits/s");
```

Comment:
We have to keep both symbols `?` because they have different roles:
* `(?:regex)` → [non-capturing parentheses group](http://www.regular-expressions.info/refcapture.html) (because you don't use this group in your subsequent code)
* `regex?`    → [greedy quantifier](http://www.regular-expressions.info/refcapture.html)